src/mono/mono.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm a bit confused about why we have this extra `nuspec` file. Are we publishing this as an extra package? Don't all the assets also get bundled into `Microsoft.AspNetCore.Blazor.Build` due to your other changes in this PR?

Reply:
nah, I just need to kill this